presto-main/src/test/java/io/prestosql/operator/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -275,16 +289,16 @@ else if (matchRate > 1) {
     }
 
     @Benchmark
-    public JoinBridgeManager<PartitionedLookupSourceFactory> benchmarkBuildHash(BuildContext buildContext)
+    public void benchmarkBuildHash(BuildContext buildContext)
```

Comment:
this need to return value for benchmarking purposes (otherwise JIT could eliminate calls).